wa/commands/runstatus.py

```diff
@@ -0,0 +1,298 @@
+import os
+import logging
+from datetime import datetime, timedelta
+
+import colorama
+
+from wa.framework.configuration.core import Status
+from wa import Command, settings
+from wa.framework.output import RunOutput, discover_wa_outputs
+from wa.framework.exception import NotFoundError
+from wa.utils.terminalsize import get_terminal_size
+from wa.utils.doc import format_column
+from wa.utils.log import COLOR_MAP, RESET_COLOR
+
+class StatusCommand(Command):
+
+    name = 'runstatus'
+    description = '''
+    Monitor ongoing runs and provide information on their progress.
```

Comment:
This should probably be phrased in the singular form as well since we currently only support one run at a time.